sos-centar-api/SosCentar.BusinessLogic/Services/UserService.cs

```diff
@@ -19,5 +22,81 @@ public bool ValidateUserCredentials(string email, string password)
 		{
 			return _userRepository.ValidateUserExists(email, _securityService.HashPassword(password));
 		}
+
+		public User GetUserIfValid(string email, string password)
+		{
+			return _userRepository.GetUserIfValid(email, _securityService.HashPassword(password));
+		}
+
+		public IEnumerable<ResponseUserDto> GetAllUsers()
+		{
+			var users = _userRepository.GetAll();
+			var userDtos = new List<ResponseUserDto>();
+
+			foreach (var user in users)
+			{
+				var userDto = new ResponseUserDto()
+				{
+					Email = user.Email,
+					FirstName = user.FirstName,
+					LastName = user.LastName,
+					IsAdmin = user.IsAdmin
+				};
+				userDtos.Add(userDto);
+			}
+
+			return userDtos;
```

Comment:
this can be replaced with simple linq query. something like this:
```c#
return _userRepository.GetAll().Select(user => new ResponseUserDto
{
  Email = user.Email,
  FirstName = user.FirstName,
  LastName = user.LastName,
  IsAdmin = user.IsAdmin
}
```